windows/visual-studio/packer.yaml

```diff
@@ -114,7 +114,8 @@ provisioners:
     elevated_password: "{{.WinRMPassword}}"
     scripts:
       - "windows/provision-scripts/update-pwsh.ps1"
-      - "windows/provision-scripts/install-windows-updates.ps1"
+      # win update took more than 5 hour which is the limit for build run
+      # - "windows/provision-scripts/install-windows-updates.ps1"
```

Comment:
This took a lot of time and still did not complete so going to uncomment it.

```
,,ui,say,==> amazon-ebs: Provisioning with powershell script: windows/provision-scripts/install-windows-updates.ps1
```

<img width="1063" alt="Screenshot 2024-07-25 at 17 27 33" src="https://github.com/user-attachments/assets/3c168baf-d14b-4503-9e42-afb808b6045b">